### PR TITLE
Bug 1905850: add a new option for checking a subresource to oc adm policy who-can

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: pkg/cli/admin/policy/who_can.go [hunk 1]
```diff
@@ -63,6 +65,7 @@ func NewCmdWhoCan(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobr
 	}
 
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If true, list who can perform the specified action in all namespaces.")
+	cmd.Flags().StringVar(&o.subresource, "subresource", o.subresource, "SubResource such as log or scale")
```
[the remark this answers is not in the document]
Comment (atiratree) [reply pkg/cli/admin/policy/who_can.go:68]: this is inspired after kubectl :) 

```
> kubectl auth can-i -h

    --subresource='':
	SubResource such as pod/log or deployment/scale
```

not sure which is better...

**Pull request merged**